x-pack/legacy/plugins/task_manager/index.ts

```diff
@@ -55,15 +55,15 @@ export function taskManager(kibana: any) {
       const schema = new SavedObjectsSchema(this.kbnServer.uiExports.savedObjectSchemas);
       const serializer = new SavedObjectsSerializer(schema);
       const { callWithInternalUser } = server.plugins.elasticsearch.getCluster('admin');
-      const savedObjectsRepositoryWithInternalUser = server.savedObjects.getSavedObjectsRepository(
+      const savedObjectsRepository = server.savedObjects.getSavedObjectsRepository(
```

Comment:
I'm thinking this file is marked outdated because this was a revert commit when I moved away from task manager using encrypted saved objects. PR no longer changes files within the task_manager folder.